pkg/kubelet/podresourcesclient.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2024 ScyllaDB.
```

Comment:
this should probably go to third_party dir unless we want to differ

Reply:
third party contains code copy pasted from other repo where we couldn't vendor it. This one was manually written by me.

Reply:
ok, it seemed similar to kube, but there probably aren't many ways to do it